closed/autoconf/custom-hook.m4

```diff
@@ -194,7 +194,7 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
 AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
   OPENJDK_SHA=`git -C $SRC_ROOT rev-parse --short HEAD`
-  OPENJDK_TAG=`git -C $SRC_ROOT describe --abbrev=0 --tags "${OPENJDK_SHA}"`
+  OPENJDK_TAG=`git -C $SRC_ROOT describe --abbrev=0 --tags --match "jdk9-*" "${OPENJDK_SHA}"`
```

Comment:
Should this be `jdk-9` based on the tags in OpenJDK?

Reply:
Yes, the pattern should be "jdk-9*" - I'll fix that. Thanks, @DanHeidinga.